utils/CMakeLists.txt

```diff
@@ -2,3 +2,51 @@ cmake_minimum_required (VERSION 3.10)
 project (HDF5_UTILS C)
 
 add_subdirectory (mirror_vfd)
+
+#-----------------------------------------------------------------------------
+# Add the vds_elink_compat_vol executable
+#-----------------------------------------------------------------------------
+
+set (vds_elink_compat_vol_SOURCES ${HDF5_UTILS_SOURCE_DIR}/vds_elink_compat_vol.c)
+add_executable (vds_elink_compat_vol ${vds_elink_compat_vol_SOURCES})
+target_include_directories (vds_elink_compat_vol PRIVATE "${HDF5_UTILS_DIR};${HDF5_SRC_DIR};${HDF5_TEST_SRC_DIR};${HDF5_SRC_BINARY_DIR};$<$<BOOL:${HDF5_ENABLE_PARALLEL}>:${MPI_C_INCLUDE_DIRS}>")
+if (NOT BUILD_SHARED_LIBS)
+  TARGET_C_PROPERTIES (vds_elink_compat_vol STATIC)
+  target_link_libraries (vds_elink_compat_vol PRIVATE ${HDF5_LIB_TARGET} ${HDF5_TEST_LIB_TARGET})
+else ()
+  TARGET_C_PROPERTIES (vds_elink_compat_vol SHARED)
+  target_link_libraries (vds_elink_compat_vol PRIVATE ${HDF5_LIBSH_TARGET} ${HDF5_TEST_LIBSH_TARGET})
+endif ()
+set_target_properties (vds_elink_compat_vol PROPERTIES FOLDER utils)
+set_global_variable (HDF5_UTILS_TO_EXPORT "${HDF5_UTILS_TO_EXPORT};vds_elink_compat_vol")
+set (H5_DEP_EXECUTABLES ${H5_DEP_EXECUTABLES} vds_elink_compat_vol)
```

Comment:
Why is this being tagged for installation? The function would be more useful as a public API.